src/sdk/python/src/iotea/core/talent_func.py

```diff
@@ -17,8 +17,8 @@
 from .util.talent_io import TalentInput, TalentOutput
 
 class FunctionTalent(Talent):
-    def __init__(self, tid, connection_string):
-        super(FunctionTalent, self).__init__(tid, connection_string)
+    def __init__(self, id, connection_string, max_threadpool_workers=1024):
```

Comment:
Idea: We could move this default value to constants.py.
(Same for "default" timeouts)

Can be done with an upcoming pull request

Reply:
Yes, we can move out these magic numbers into constants.